src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Only box-sizing property allowed for `*` selector. Move this properties to body selector